packages/styles/components/Stepper.module.scss

```diff
@@ -58,7 +58,7 @@
     width: var(--space-xs);
   }
 
-  @include above($screen-lg) {
+  @include above(desktopL) {
```

Comment:
How this works?

Reply:
You can check [rupture](https://www.npmjs.com/package/rupture-sass) 😊